client/time.h

```diff
@@ -21,16 +21,28 @@
 extern "C" {
 #endif
 
-extern xdag_time_t g_xdag_era;
```

Comment:
Just one question: what was wrong with name "xdag_time_t"?

Reply:
just that: function name are ugly or may not clear, like xdag_get_xdag_timestamp or xdag_time_to_string and time_to_string(cannot add xdag prefix here to avoid misunderstanding, for example)